eml_parser/regexes.py

```diff
@@ -37,7 +37,31 @@
 #   - do not use a fixed list of TLDs but rather \w
 #   - only check for URLs with scheme
 #   - modify the end marker to allow any acceptable char according to the RFC3986
-url_regex_simple = re.compile(r'''(?i)\b(?:https?|ftps?):(?:/{1,3}|[a-z0-9%])(?:[^\s()<>{}\[\]]+|\([^\s()]*?\([^\s()]+\)[^\s()]*?\)|\([^\s]+?\))+(?:[\w\-._~%!$&'()*+,;=:/?#\[\]@]+)''')
+
+if re.__name__ == 're2':
+    url_regex_simple = re.compile(r'''
```

Comment:
Quickly testing some stuff I noticed that the following URL does not match with your modified regex:
http://example.com/?

Could you please have a look at that ?

-> FYI I tested with the non re2 version

Reply:
The regex should match `http://example.com/?`, but urllib.parse removes the trailing `?`

Reply:
At least on regex101 it does not work, see https://regex101.com/r/om1AXp/1

Strangely "https://www.example.com?" matches

Reply:
I had `?` in the negated character class. I had removed the `?` for #69, but had not merged that into this.

Reply:
looks good